docs/index.html

```diff
@@ -3292,8 +3292,8 @@ <h4 id="_load_xml_introduction"><a class="link" href="#_load_xml_introduction">L
 <p>Many existing (enterprise) applications, endpoints and files use XML as data exchange format.</p>
 </div>
 <div class="paragraph">
-<p>To make these datastructures available to Cypher, you can use <code>apoc.load.xml</code>.
-It takes a file or http URL and parses the XML into a map datastructure.</p>
+<p>To make these Data Structures available to Cypher, you can use <code>apoc.load.xml</code>.
```

Comment:
We don't tend to capitalise words like this in our docs, so it would need to be: data structures

Reply:
ok.. thanks